crates/proof-of-sql/src/base/database/expression_evaluation.rs

```diff
@@ -74,6 +75,10 @@ impl<S: Scalar> OwnedTable<S> {
         let column = self.evaluate(expr)?;
         match op {
             UnaryOperator::Not => Ok(column.element_wise_not()?),
+            // Handle unsupported unary operators
+            _ => Err(ExpressionEvaluationError::UnsupportedUnaryOperator {
```

Comment:
Actually maybe we can just use `ExpressionEvaluationError::Unsupported`.

Reply:
Done